S/SFML/build_tarballs.jl

```diff
@@ -88,9 +68,11 @@ dependencies = [
     Dependency("FLAC_jll"),
     Dependency("FreeType2_jll"),
     Dependency("libvorbis_jll"),
-    Dependency("Xorg_libXrandr_jll"),
-    Dependency("Xorg_libX11_jll"),
-    Dependency("OpenAL_jll")
+    BuildDependency("Xorg_libXrandr_jll"),
+    BuildDependency("Xorg_libX11_jll"),
```

Comment:
Why?  These packages have runtime libraries
```suggestion
    Dependency("Xorg_libXrandr_jll"),
    Dependency("Xorg_libX11_jll"),
```

Reply:
Oops!